plugins/nodes/src/js/components/NodesTable.tsx

```diff
@@ -159,124 +166,135 @@ export default class NodesTable extends React.Component<
-                sortHandler={this.handleSortClick.bind(null, "tasks")}
-                sortDirection={sortColumn === "tasks" ? sortDirection : null}
-                textAlign="right"
-              />
-            }
-            cellRenderer={tasksRenderer}
-          />
-
-          <Column
-            header={
-              <SortableHeaderCell
-                columnContent={<Trans render="span">CPU</Trans>}
-                sortHandler={this.handleSortClick.bind(null, "cpu")}
-                sortDirection={sortColumn === "cpu" ? sortDirection : null}
-              />
-            }
-            cellRenderer={cpuRenderer}
-          />
-
-          <Column
-            header={
-              <SortableHeaderCell
-                columnContent={<Trans render="span">Mem</Trans>}
-                sortHandler={this.handleSortClick.bind(null, "mem")}
-                sortDirection={sortColumn === "mem" ? sortDirection : null}
-              />
-            }
-            cellRenderer={memRenderer}
-          />
-
-          <Column
-            header={
-              <SortableHeaderCell
-                columnContent={<Trans render="span">Disk</Trans>}
-                sortHandler={this.handleSortClick.bind(null, "disk")}
-                sortDirection={sortColumn === "disk" ? sortDirection : null}
-              />
-            }
-            cellRenderer={diskRenderer}
-          />
-
-          <Column
-            header={
-              <SortableHeaderCell
-                columnContent={<Trans render="span">GPU</Trans>}
-                sortHandler={this.handleSortClick.bind(null, "gpu")}
-                sortDirection={sortColumn === "gpu" ? sortDirection : null}
-              />
-            }
-            cellRenderer={gpuRenderer}
-          />
-
-          <Column
-            header={<span title="Spacing" />}
-            cellRenderer={spacingRenderer}
-            width={spacingSizer}
-          />
-        </Table>
+        <NodesTableWrapper data={data.slice()}>
+          <Table data={data.slice()}>
+            <Column
+              header={
+                <SortableHeaderCell
+                  columnContent={<Trans render="span">Host</Trans>}
+                  sortHandler={this.handleSortClick.bind(null, "host")}
+                  sortDirection={sortColumn === "host" ? sortDirection : null}
+                />
+              }
+              cellRenderer={ipRenderer}
+            />
+
+            <Column
+              header={
+                <SortableHeaderCell
+                  columnContent={<Trans render="span">Health</Trans>}
+                  sortHandler={this.handleSortClick.bind(null, "health")}
+                  sortDirection={sortColumn === "health" ? sortDirection : null}
+                />
+              }
+              cellRenderer={healthRenderer}
+            />
+
+            <Column
+              header={
+                <HeaderCell>
```

Comment:
I still think this should be sortable.

Reply:
It is unfortunately not sortable since we only wrap the table and add the information only there but the sorting happens earlier we will address this at a later point.